### DISCUSSION TITLE
Problems with dumping save from Secret of Evermore

### DISCUSSION THREAD

**Discussion by SecretofNevermore** [sanni/cartreader#434] [General Chat]
Hey all, just acquired my cart reader recently and while I'm enjoying it I have run into a problem.

I've been able to dump the ROM from secret of evermore without much issue (although recently I dumped it after flashing 9.1 and got a checksum error) 

But my major issue is the sram or save dump. It dumps a file but no emulator loads it. They ignore it entirely even though I've made sure the names match up and the pathways are all correct. I've managed to dump roms and saves from Chrono Trigger and Final Fantasy 6 without a problem. They play perfectly and the saves are accessible. 

The game itself boots up perfectly on my SNES and the save file is intact. I'm assuming I have done something wrong but I don't know what. I know Chrono Trigger is 32mb and secret of evermore is 24 but idk if that has any bearing on this. 

Attached is a picture of the error I got this time dumping the ROM. 

Any help is appreciated. 
![PXL_20220724_201948944](https://user-images.githubusercontent.com/109926800/180670217-f949ac84-9b16-498c-949b-9285270dc72b.jpg)

**Comment by SAVE-THE-HERO** [comment]
Hello.
It looks does not appear to be a device fault.
I do not have Secret of Evermore and cannot verify this.
So Try it once, The Cart Reader is very picky when it comes to contact cleanliness.
Disassemble the cartridge and you can try clean the cart edge connector of the game with some rubbing alcohol.
Thank you

**Reply by SecretofNevermore** [reply]
Thank you. It occured to me last night to do this and it all works now. I appreciate the reply.

**Comment by sanni** [comment]
Dip a q-tip into rubbing alcohol and go over the gold contacts of your game until there is no black stuff on the q-tip anymore. 
Secret of Evermore uses a standard HiRom PCB so it should dump correctly.

**Reply by SecretofNevermore** [reply]
Funnily enough I was wracking my brain thinking I couldn't possibly have flashed the Arduino poorly since it worked with all my other games, so of course Occam's razor, it would be the simple solution that I didn't think of. 

So I grabbed the IPA 99 and scrubbed thoroughly, everything works now. 

In my rather poor defense, the game works perfectly in my SNES so I didn't think to clean the cartridge contacts.
